src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -827,23 +875,25 @@ private void CommitCore(IUIThreadOperationContext operationContext, bool preview
 
                 if (newDocument.SupportsSyntaxTree)
                 {
-                    // We pass CancellationToken.None here because we don't have a usable token to pass. The IUIThreadOperationContext
-                    // passed here as a cancellation token, but the caller in CommitCore has already turned off cancellation
-                    // because we're committed to the update at this point. If we ever want to pass cancellation here, we'd want to move this
-                    // part back out of this method and before the point where we've already opened a global transaction.
-                    var root = newDocument.GetSyntaxRootSynchronously(CancellationToken.None);
+                    // We pass CancellationToken.None here because we don't have a usable token to pass. The
+                    // IUIThreadOperationContext passed here as a cancellation token, but the caller in CommitCore has
+                    // already turned off cancellation because we're committed to the update at this point. If we ever
+                    // want to pass cancellation here, we'd want to move this part back out of this method and before
```

Comment:
comment seems off - says we pass None, but we just pass in the callers token

Reply:
fixed.